src/csfloat/content_script.ts

```diff
@@ -651,11 +652,38 @@ async function adjustItem(container: Element, isPopout = false) {
                 await addListingAge(container, apiItem);
                 await caseHardenedDetection(container, apiItem, true);
                 await addFadePercentages(container, apiItem);
+                await addFloatColoring(container, apiItem);
             }
         }, 500);
     }
 }
 
+async function addFloatColoring(container: Element, item: CSFloat.ListingData) {
```

Comment:
Is this the exact same function as in `skinport/content_script.ts`? 

If so, please export this function to `util/helperfunctions.ts` and reuse it ([why this improves code quality](https://sourcemaking.com/refactoring/form-template-method)).
Since most likely the parameters are different, just minimize it to the minimal necessary attributes (e.g. [Demeter's Law](https://en.wikipedia.org/wiki/Law_of_Demeter)).

Reply:
Made a refactor but keeping the addFloatColoring to get the htmlElement to edit and set the color, I think it's ok for code readability